adm_program/installation/update.php

```diff
@@ -102,11 +102,27 @@
 $gLanguageData = new LanguageData($gSettingsManager->getString('system_language'));
 $gL10n = new Language($gLanguageData);
 
+if (!is_executable(ADMIDIO_PATH . FOLDER_DATA) || !is_writable(ADMIDIO_PATH . FOLDER_DATA))
+{
+    try
+    {
+        FileSystemUtils::chmodDirectory(ADMIDIO_PATH . FOLDER_DATA, 0777);
+    }
+    catch (\RuntimeException $exception)
+    {
+        // TODO
```

Comment:
We should leave a message that the user must set the rights to **adm_my_files**.